src/mono/browser/runtime/exports.ts

```diff
@@ -42,22 +40,12 @@ function initializeExports (globalObjects: GlobalObjects): RuntimeAPI {
         instantiate_asset,
         jiterpreter_dump_stats,
         forceDisposeProxies,
-        instantiate_segmentation_rules_asset,
 
     };
     if (WasmEnableThreads) {
         rh.dumpThreads = mono_wasm_dump_threads;
         rh.mono_wasm_print_thread_dump = () => tcwraps.mono_wasm_print_thread_dump();
     }
-    if (loaderHelpers.config.globalizationMode === GlobalizationMode.Hybrid) {
-        rh.stringToUTF16 = stringToUTF16;
```

Comment:
is any of the string functions now un-used ?

Reply:
`instantiate_segmentation_rules_asset` is deleted from `src/mono/browser/runtime/assets.ts`, I cannot find any other occurrence that is left, so I guess it's okay.
`stringToUTF16` is used but it's taken from the main module. This one that is deleted in the quoted code is a forwarded method to hybrid globalization module. During removal of hybrid code, we also removed the hybrid module, that's why this line is fine.

Reply:
"file-based SDK tests" what is a file-based test?

Reply:
but for .js file